presto-main/src/main/java/com/facebook/presto/sql/gen/RowExpressionCompiler.java

```diff
@@ -107,9 +106,6 @@ public BytecodeNode visitCall(CallExpression call, Context context)
                         // (SWITCH <expr> (WHEN <expr> <expr>) (WHEN <expr> <expr>) <expr>)
                         generator = new SwitchCodeGenerator();
                         break;
-                    case TRY:
-                        generator = new TryCodeGenerator(preGeneratedExpressions.getTryMethodMap());
```

Comment:
Can remove `TryCodeGenerator`  since it should be no longer used ?

Reply:
@wenleix Thanks! I've removed TryCodeGenerator and some related leftovers. I'm also thinking of refactoring `LambdaAndTryExpressionExtractor` to `LambdaExpressionExtractor`. Does this sound good?

Reply:
That sounds reasonable to me. But maybe we should do it in a separate PR, as we might also want to rename PreGeneratedExpression and other stuffs as well. And doing all renaming in a single PR sounds easier to me :D

@haozhun any thoughts?

Reply:
I agree that they should belong to another PR :)